### PR TITLE
Fix for issue #229

### PR DESCRIPTION
**Pull request by jfiedler**
Fix for issue #229 (watch not working if config format == FILES)

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@jfiedler Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-consul&pullRequestId=230)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-consul&pullRequestId=230) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/faq) for frequently asked questions.

**Comment by spencergibb** [comment]
LGTM pending CLA signing.

File: spring-cloud-consul-config/src/main/java/org/springframework/cloud/consul/config/ConfigWatch.java [hunk 1]
```diff
@@ -70,7 +72,9 @@ public void start() {
 	public void watchConfigKeyValues() {
 		if (this.running.get()) {
 			for (String context : this.contexts) {
-				if (!context.endsWith("/")) {
+
+				// turn the context into a Consul folder path (unless our config format are FILES)
```
Comment (spencergibb) [inline spring-cloud-consul-config/src/main/java/org/springframework/cloud/consul/config/ConfigWatch.java]: good comment!

**Comment by spencergibb** [comment]
Waiting for @jfiedler to sign the cla

**Comment by pivotal-issuemaster** [comment]
@jfiedler Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-consul&pullRequestId=230)!

**Pull request merged**